tests/web3/batch_partial_notification.out.json

```diff
@@ -0,0 +1 @@
+[{"jsonrpc":"2.0","id":1,"result":"28346"}]
```

Comment:
I'm confused why this doesn't have two responses, is it intended?

Reply:
Looks likee there are two requests in the file above.

Reply:
This is the correct behavior. One of the requests is called a "notification" because it does not have an id field and thus it does not have a corresponding response.